docs/en/index.md

```diff
@@ -6,13 +6,21 @@
 
 The module is set up in the `Settings` section of the CMS, see the [User guide](userguide/index.md).
 
-### Reminder emails
+### Email notifications
 
-In order for the contentreview module to send emails, you need to *either*:
+In order for the contentreview module to send overdue review and reminder email notifications, you need to *either*:
```

Comment:
Is is `either`? or do you need to do all of the following?

Reply:
You need to do both if you want both reminder and review notifications. I guess its more "either" as the choice is with the site admin